k8s/notify-pending-prs.yaml.j2

```diff
@@ -0,0 +1,25 @@
+replicaCount: 1
```

Comment:
This file needs to be renamed to a `.yml.j2` extension. The extra `a` is going to cause the deploy workflow to fail since it's looking for `yml`

Reply:
fixed